website/src/main/tut/docs/rules/LeakingSealed.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: docs
+title: LeakingSealed
+---
+
+# LeakingSealed
+
+_Since 0.5.8_
+
+This rule ensures that descendants of a sealed types are final or sealed. 
+
+It disallows the following situation:
+
+`file 1:`
+```scala
+sealed trait t
+trait a extends t
+class c extends t
```

Comment:
add leading comment where the error message would appear `// error: leaking sealed`

it would also help to have a positive example, where there's a comment `// ok, because final does not leak sealed`